### PR TITLE
Created merge_sorted_lists and tests for that

### PR DESCRIPTION
**Pull request by lindsaybolz**

### CODE REVIEW

File: functions/merge_sorted_lists.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 def merge_sorted_lists(l1, l2):
-    pass
+    merged = l1 + l2
```
Comment (WinsEdwards) [inline functions/merge_sorted_lists.py:2]: really clean approach. how could you refactor if you wanted to merge into a larger sorted list?

File: tests/merge_sorted_lists_test.py [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+    l2 = [1]
+
+    # Act
+    answer = merge_sorted_lists(l1, l2)
+
+    # Assert
+    assert answer == [1, 2]
+
+
+def test_for_1_2_3_and_4_5_6():
```
Comment (WinsEdwards) [inline tests/merge_sorted_lists_test.py:29]: These tests do a great job with nominal cases. What about edge cases such as if the two lists have numeric overlap?

File: tests/merge_sorted_lists_test.py [hunk 3]
```diff
@@ -1,2 +1,38 @@
...
+    # Arrange
+    l1 = [1, 2, 3]
+    l2 = [4, 5, 6]
```
Comment (WinsEdwards) [inline tests/merge_sorted_lists_test.py:32]: imagine if the lists are: 
l1 = [1, 2, 3, 4]
l2 = [3, 4, 5, 6]
